test/test-initial-properties.js

```diff
@@ -144,6 +167,15 @@ it('anonymous initial properties (predefined attributes)', () => {
   assert(el.shadowRoot.textContent === 'ONE, TWO');
 });
 
+it('compound initial properties are not shared accross element instances', () => {
+  const el1 = document.createElement('test-element-compound');
+  const el2 = document.createElement('test-element-compound');
+  document.body.append(el1, el2);
+  assert(el1.shadowRoot.textContent === 'foo');
+  assert(el2.shadowRoot.textContent === 'foo');
+  assert(el1.compound !== el2.compound);
```

Comment:
This is the behavior that you're looking for, right?

Reply:
Yep — it looks like something is wrong on my end. I'll see if I can isolate a reproduction case

Reply:
As discussed, I think the confusion here is that we haven't released the changes that fix the issue you're seeing! I'm going to merge this and then release.